docs/how_to_integrate.md

```diff
@@ -2,3 +2,21 @@
 
 *TBD*
 
+## Basic Integration
+
+-   Updating repositories.bzl
+    -   Layering you dependencies.
+-   Creating setup/*rules_mine*.bzl
+    -   Bringing in more dependencies
+    -   Registring toolchains
+-   Adding tests
```

Comment:
Maybe mention that the need to create a presubmit config

Reply:
Done